### PR TITLE
21 create login page

### LINKED ISSUE #35: 21 - Create log-in page
- [ ] create login.html
- [ ] create login.js
- [ ] create login page sections
- [ ] include header and footer components
- [ ] style the login page through the style.css file
- [ ] save the login token into a local storage
- [ ] change the api-calls to include the tokens from the local storage

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
I'm pushing the the login page folder with these files:
- login.html
- login.js
- main-section.js
- form-section.js

the page is behaving good in terms of login and connected to the signup page
still the header and footer not added to the development branch to be add to the login page

### CODE REVIEW

File: client/styles/login.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
+.main {
```
Comment (dirk-jacobs) [inline client/styles/login.css:1]: Just to be sure that there is no impact from other style files, I would prefix the style classes with login-  eg login-form_title

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
Looks good.
Just read my remark about the class-style naming

I didn't checked if everything is working perfectly with the backend, but the basics are there and could be submitted

File: client/styles/login.css (hunk 1 above)
Comment (Moamin-AbuEwaida) [reply client/styles/login.css:1]: noted @dirk-jacobs

**Comment by Moamin-AbuEwaida** [comment]
just pushed the final commit with the latest edits and updates to the branch files

**Review by dirk-jacobs — CHANGES_REQUESTED** [review]
In code there is bcryptje in package.json there is bcrypt

**Comment by Moamin-AbuEwaida** [comment]
> In code there is bcryptje in package.json there is bcrypt


yes, when I install locally bcrypt, i've been getting broken files so I'm using bcryptjs but the rest of the team use bcrypt @dirk-jacobs 
Rayane reviewed the branch from herside- replaced bcryptjs with bcrypt in the business logic folders (login and user) then the server worked normally but now were are receiving a bug for connecting to the database server, the code was working with me when i was running it locally :thinking:

**Review by rayanejsilva — APPROVED** [review]

**Pull request merged**